binary-serialise-cbor.cabal

```diff
@@ -112,7 +112,7 @@ test-suite tests
     deepseq,
     scientific,
     vector,
-    aeson  >=0.7 && <0.9,
+    aeson  >=0.7 && <=0.9,
     half,
```

Comment:
As an outsider, I would have said that using < 0.10 is the more common way to specify this (using <= means that v0.9.0.1 won;t be considered)